project/src/components/room-info/room-info.tsx

```diff
@@ -0,0 +1,96 @@
+import classNames from 'classnames';
+
+import Badge from '../badge/badge';
+import FavoriteButton from '../favorite-button/favorite-button';
+import NewCommentForm from '../new-comment-form/new-comment-form';
+import StarRating from '../star-rating/star-rating';
+import ReviewItem from '../review-item/review-item';
+
+import { Offer } from '../../types/offer/offer';
+
+import { comments } from '../../mocks/comments';
+
+type RoomInfoProps = {
+  isPremium?: boolean;
+  offer: Offer;
+};
+
+export default function RoomInfo ({isPremium = true, offer}: RoomInfoProps): JSX.Element {
+  return (
+    <div className="property__container container">
+      <div className="property__wrapper">
+        {isPremium && <Badge str='Premium' badgeClasses={['property__mark']} />}
+        <div className="property__name-wrapper">
+          <h1 className="property__name">
+            {offer.title}
+          </h1>
+          <FavoriteButton isFavorite={offer.isFavorite} buttonType='room' />
+        </div>
+        <StarRating rating={offer.rating} starRatingType='room' />
+        <ul className="property__features">
+          <li className="property__feature property__feature--entire">
+            {offer.type}
+          </li>
+          <li className="property__feature property__feature--bedrooms">
+            {`${offer.bedrooms} Bedrooms`}
+          </li>
+          <li className="property__feature property__feature--adults">
+            {`Max ${offer.maxAdults} adults`}
```

Comment:
в реакте такая запись не нужна, можно записать так
```suggestion
            Max {offer.maxAdults} adults
```

Reply:
done